src/domains/sign/integration.tf

```diff
@@ -0,0 +1,121 @@
+module "event_hub" {
+  source                   = "git::https://github.com/pagopa/terraform-azurerm-v3.git//eventhub?ref=v4.1.7"
+  name                     = format("%s-eventhub-ns", local.project)
+  location                 = azurerm_resource_group.integration_rg.location
+  resource_group_name      = azurerm_resource_group.integration_rg.name
+  auto_inflate_enabled     = var.integration_hub.auto_inflate_enabled
+  sku                      = var.integration_hub.sku_name
+  capacity                 = var.integration_hub.capacity
+  maximum_throughput_units = var.integration_hub.maximum_throughput_units
+  zone_redundant           = var.integration_hub.zone_redundant
+
+  virtual_network_ids = [data.azurerm_virtual_network.vnet_common.id]
+  subnet_id           = module.io_sign_eventhub_snet.id
+
+  private_dns_zone_record_A_name = null
+
+  eventhubs = var.integration_hub.hubs
+
+  public_network_access_enabled = true
```

Comment:
may we have a comment explaining why this is set to true?

Reply:
This is set to true because we need our eventhub to be accessed through public network. The property name seems to be enough explicative in order not to require any comment on is effect.

The product we are integrating with requires consuming the event hub topics via public network; this choice is documented at the company wiki page relative to the integrating product.

Reply:
> The product we are integrating with requires consuming the event hub topics via public network

with a link to the wiki page looks like a nice comment :-)

Reply:
Given that this is a public repository, can we commit a link to our internal doc?

Reply:
Generalley, a safer way to link internal docs is link to a search result; this states for gdrive and/or confluence ie. https://pagopa.atlassian.net/wiki/search?text=requisiti%20accessibilit%C3%A0 (this is ok)

Reply:
Sure, you can check the link https://pagopa.atlassian.net/wiki/search?text=firma%20con%20io%20fatturazione

Reply:
can we put this inside the sources?

Reply:
Done